engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -135,9 +136,14 @@ class WorkflowManagerActor(params: WorkflowManagerActorParams)
   override def preStart(): Unit = {
     // Starts the workflow polling cycle
     timers.startSingleTimer(RetrieveNewWorkflowsKey, RetrieveNewWorkflows, Duration.Zero)
+    // Listen on subworkflow start and end events to inform our decision whether to pick up new root workflows
+    // from the workflow store.
+    context.system.eventStream.subscribe(self, classOf[SubWorkflowStart])
+    context.system.eventStream.subscribe(self, classOf[SubWorkflowEnd])
```

Comment:
I wish we had used this `eventStream` stuff from the beginning for things like the service registry actor instead of wiring it through every single Cromwell actor -_-